### PR TITLE
Add Pull Request Template

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Add a pull request template.  Modeled after CRI-O (Thanks @saschagrunert!) and
Dockers.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<!--
+Thanks for sending a pull request!
+
+Please make sure you've read and understood our contributing guidelines
+(https://github.com/containers/buildah/blob/master/CONTRIBUTING.md) as well as ensuring
+that all your commits are signed with `git commit -s`.
```
Comment (rhatdan) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is there anything to make this automatic?

Comment (TomSweeneyRedHat) [reply .github/PULL_REQUEST_TEMPLATE.md]: I'm not sure, need to dig further.  It certainly would be nice if we could pull that off.

Comment (saschagrunert) [reply .github/PULL_REQUEST_TEMPLATE.md]: Hm no I don't think we could automate this without having a write access token to the repository.

**Comment by rhatdan** [comment]
bors retry

**Pull request merged**